m3-sys/llvm/llvm9bindings/src/m3makefile

```diff
@@ -80,7 +83,7 @@ if equal (OS_TYPE, "POSIX")
 
   exec("make", "-f" , "../src/Makefile", "-k", "LLVM_INCLUDE_DIR=" & LLVM_INCLUDE_DIR, "LLVM_BUILD_INCLUDE_DIR=" & LLVM_BUILD_INCLUDE_DIR, "BUILD_DIR=" & BUILD_DIR)
 else
-  % Windows X86
+  % Both Windows amd64 and Windows X86
```

Comment:
Nit: It is just for Windows. No need to list all architectures.

Reply:
} Nit: It is just for Windows. No need to list all architectures.
I think about this... Why not?

Reply:
} Maybe no need to say "Windows".
Will be total delete? I am not author of first comment state...

Reply:
Fixed:
https://github.com/VictorMiasnikov/cm3/commit/19cfaed13c43fe431116b10811bac3ec71a35df1